mailbox/store/pom.xml

```diff
@@ -67,6 +67,10 @@
             <artifactId>james-mdn</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-data-api</artifactId>
```

Comment:
Why do we need that?

Reply:
For using `UsersRepository`... In the next PR though (#3297) I'm moving all those quotas stuff into a mailbox-tools module. Still thanks for the comment cause I think I forgot to remove it there, but is it ok to fix it in the other one? with all classes moving out it's easier.

@chibenwa proposed as well to implement a method in QuotaRootResolver to list all quotaRoots of all users (which would allow to remove it from the service). I didn't have the time to investigate enough on this, as adding this in QuotaRootResolver seems to change tricky stuff in some test classes. But I can give it a shot later

Reply:
Still needed?

Reply:
sorry but I can't let that merged. Could you pick the part that fixes this issue in that PR please?

Reply:
I need it as I need to use UsersRepository....

Reply:
Ok with @chibenwa we extracted the service in its own module, remving that dependancy from mailbox-store